### PR TITLE
[MIG] procurement module migration scripts

### PR DESCRIPTION
**Pull request by cubells**
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

 * completed analysys
 * added analysis_work file

### CODE REVIEW

File: addons/procurement/migrations/9.0.1.0/openupgrade_analysis.txt [hunk 1]
```diff
@@ -1,10 +1,16 @@
 ---Fields in module 'procurement'---
-procurement  / procurement.order        / message_follower_ids (many2many): not a function anymore
-procurement  / procurement.order        / message_follower_ids (many2many): relation is now 'mail.followers' ('res.partner')
-procurement  / procurement.order        / message_follower_ids (many2many): type is now 'one2many' ('many2many')
-procurement  / procurement.order        / product_uos (many2one)        : DEL relation: product.uom
-procurement  / procurement.order        / product_uos_qty (float)       : DEL 
-procurement  / procurement.order        / website_message_ids (one2many): NEW relation: mail.message
+procurement  / procurement.order        / message_follower_ids (many2many)      : not a function anymore
```
Comment (pedrobaeza) [inline addons/procurement/migrations/9.0.1.0/openupgrade_analysis.txt]: Why have you changed this file?

**Comment by pedrobaeza** [comment]
You have to update documentation in https://github.com/OCA/OpenUpgrade/blob/9.0/openerp/openupgrade/doc/source/modules80-90.rst

File: addons/procurement/migrations/9.0.1.0/openupgrade_analysis.txt (hunk 1 above)
Comment (cubells) [reply addons/procurement/migrations/9.0.1.0/openupgrade_analysis.txt]: To add some analysis of fields/views that are missing in initial analysis.

Comment (pedrobaeza) [reply addons/procurement/migrations/9.0.1.0/openupgrade_analysis.txt]: Two things about this:

* Don't change lines although the indentation is not so good for not increasing the diff.
* Why they are not included? Maybe we should improve the analysis method. Anyway, don't include here that extra lines, but it _work file.

File: addons/procurement/migrations/9.0.1.0/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+procurement  / procurement.order        / message_partner_ids (many2many)       : NEW relation: res.partner
+# New field ---> Nothing to do
+
+procurement  / procurement.order        / message_summary (text)                : DEL
+# Nothing to do
+
+procurement  / procurement.order        / message_unread_counter (integer)      : NEW
+# New field ---> Nothing to do
+
+procurement  / procurement.order        / product_uos (many2one)                : DEL relation: product.uom
```
Comment (pedrobaeza) [inline addons/procurement/migrations/9.0.1.0/openupgrade_analysis_work.txt:25]: If this dissapears, maybe now `product_uos` and `product_uos_qty` of the sales orders are put directly in `product_uom` and `product_uom_qty`. If so, then you should include a script for copying "uos" values (in pre-migration) to "uom" when "uos" is present. Please check the behavior in v8 and v9 when you put sales units and tell me back.

Comment (cubells) [reply addons/procurement/migrations/9.0.1.0/openupgrade_analysis_work.txt:25]: Not.

product_uos and product_uos_qty dissapear:
https://github.com/OCA/OpenUpgrade/blob/9.0/addons/sale/sale.py#L557
and product_uom is setted with product_uom and product_uom_qty.


I'm right?

Comment (pedrobaeza) [reply addons/procurement/migrations/9.0.1.0/openupgrade_analysis_work.txt:25]: OK, correct, then nothing to do on this.

File: openerp/openupgrade/doc/source/modules80-90.rst [hunk 3]
```diff
@@ -348,7 +348,7 @@ Status :
 +-----------------------------------+-----------------------------------+
 |pos_restaurant                     |                                   |
 +-----------------------------------+-----------------------------------+
-|procurement                        |                                   |
+|procurement                        | Done                              |
```
Comment (pedrobaeza) [inline openerp/openupgrade/doc/source/modules80-90.rst]: s/Done/Nothing to do

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**